packages/dd-trace/src/format.js

```diff
@@ -58,6 +58,10 @@ function extractTags (trace, span) {
       case 'resource.name':
         addTag(trace, {}, map[tag], tags[tag])
         break
+      // HACK: remove when backend supports numeric status code
```

Comment:
`s/backend/agent/`

Reply:
Doesn't matter, it's not supported by something along the way.

Reply:
Yeah, it doesn't, just being pedantic ;P